examples/recently-viewed-items/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "main": "index.html",
   "scripts": {
-    "build": "parcel build index.html",
+    "build": "parcel build index.html --dist-dir ./dist",
```

Comment:
Why do we need this?

Reply:
On my side, running the actual `yarn build` in `parcel 2.0.0-beta.2` will output the build in the root folder.

Could you confirm this is also the case for you? If so, I'll update the other examples

Reply:
It builds it in the right folder for me.

What command did you run and from where?

Reply:
From the example folder (the root), with `yarn build`, `yarn start` works though.

Reply:
Indeed!